edit-instrument-recordings-links.user.js

```diff
@@ -0,0 +1,58 @@
+// ==UserScript==
+// @name		MusicBrainz: Add recording edit links to instrument pages
+// @description Direct links to the recording edit page are added to instruments' recordings pages.
+// @version		2019.4.2.1
+// @author		Nicolás Tamargo
+// @license     X11
+// @downloadURL https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/edit-instrument-recordings-links.user.js
+// @updateURL   https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/edit-instrument-recordings-links.user.js
+// @include     *://*.musicbrainz.org/instrument/*/recordings*
```

Comment:
You also want to include `https://musicbrainz.org`:
```
// @include     *://musicbrainz.org/instrument/*/recordings*
```

Reply:
Oh, duh. Done.